README.md

```diff
@@ -9,14 +9,7 @@ use Consul with Kubernetes, please see the
 [Consul and Kubernetes documentation](https://www.consul.io/docs/platform/k8s/index.html).
 
 ## Prerequisites
-
-To use the charts here, [Helm](https://helm.sh/) must be installed in your
-Kubernetes cluster. Setting up Kubernetes and Helm and is outside the scope
-of this README. Please refer to the Kubernetes and Helm documentation.
-
-The versions required are:
-
-  * **Helm 2.10+** - This is the earliest version of Helm tested. It is possible
+  * **Helm 2.10+ or Helm 3.0+** - This is the earliest version of Helm tested. It is possible
     it works with earlier versions but this chart is untested for those versions.
```

Comment:
Do we need to add tests for both versions of helm if we're saying that?

Reply:
I've been using Helm 3 extensively so I don't think so. We should switch to Helm 3 for our template tests but I don't think that needs to block this.

Reply:
Yeah me too, but I think this refers to automatic testing. I can't guarantee that I will test every PR or every change myself. I think if we're not testing both, we shouldn't say that we test it. Maybe we could just rephrase this to not say which versions we're testing with and say that it *works* with these versions of Helm.

Reply:
Ahh gotcha, yeah will reword.

Reply:
I just changed it to "Helm 2.10+ or Helm 3.0+". What do you think?

Reply:
Nice!